sherpa-ncnn/csrc/recognizer.cc

```diff
@@ -131,16 +205,24 @@ class Recognizer::Impl {
   void DecodeStream(Stream *s) const {
     int32_t segment = model_->Segment();
     int32_t offset = model_->Offset();
+    bool has_context_graph = false;
+
+    if (!has_context_graph && s->GetContextGraph())
+      has_context_graph = true;
```

Comment:
```suggestion
    if (!has_context_graph && s->GetContextGraph()) {
      has_context_graph = true;
     }
```

Please use `{}` even if it has only a single statement for consistency.

Reply:
done